near/omni-types/src/evm/events.rs

```diff
@@ -14,11 +14,11 @@ sol! {
         address indexed sender,
         address indexed tokenAddress,
         uint128 indexed nonce,
-        string token,
         uint128 amount,
         uint128 fee,
         uint128 nativeTokenFee,
-        string recipient
+        string recipient,
+        string message
```

Comment:
FinTransfer in this file should be updated as well:
string token -> address tokenAddress

Reply:
f1a089f9b1aeb39217d2882c1d440330a0d67975